Training/PythonWebServer/index.html

```diff
@@ -9,6 +9,33 @@
     <script src="/resources/index.js" type="text/javascript"></script>
 </head>
 <body>
-    <h1>Hello world!</h1>
+	<div class='wrapper'>
+        <button class="button" onclick="loadDoc()">UPDATE</button>
+
+    	<div class="display">
+        Time:
+            <p id = "Time"></p> 
+
+        <br><br><br>
+    	</div>
+
+    	<div class="display">
+        Total CPUs: 
+        <p id = "totalCPU"></p>
+        <br> <br>
+    	CPU Usage:
+            <p id = "cpuUsage"></p> 
+    	</div>
+
+    	<div class="display">
+    	Total RAM:
+            <p id = "totalRAM"></p>
+    		<br><br>
+    	Used RAM:
```

Comment:
In general it's bad practice to have text not directly in it's own tag, it makes it hard to style them individually.